src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -466,3 +466,17 @@ function Get-ComputePublisherLocation
 {
      Get-ResourceProviderLocation "Microsoft.Compute/locations/publishers";
 }
+
+function Get-SubscriptionIdFromResourceGroup
+{
+      param ([string] $rgname)
+
+	  $rg = Get-AzureRmResourceGroup -ResourceGroupName $rgname;
+
+	  $rgid = $rg.ResourceId;
+
+	  $first = $rgid.IndexOf('/', 1);
+	  $last = $rgid.IndexOf('/', $first + 1);
```

Comment:
Could you please add a comment for the resource ID format? It would make the code more clear.